docs/developer-guide/shadertools/writing-glsl-shaders.md

```diff
@@ -1,41 +1,63 @@
-# GLSL Support
+# Writing GLSL Shaders
 
-## GLSL Versions
+## GLSL Versions and Compatibility
 
-> The GLSL versions discussed here are the OpenGL ES versions GLSL 1.00 ES and GLSL 3.00 ES, which distinguishes them from other GLSL versions for the non ES versions of OpenGL. For simplicity, we will often refer to the version numbers without the `ES` suffix.
+> The GLSL versions discussed here are the two GLSL versions supported by WebGL2, namely GLSL 3.00 ES and GLSL 1.00 ES. The ES suffix indicates that they are related to the OpenGL ES standards used by WebGL. For simplicity, we will often refer to the GLSL version numbers without the `ES` suffix.
 
-While WebGL1 support GLSL 1.00 only, WebGL2 introduces the choice of writing shaders in GLSL version 3.00.
+While WebGL1 supports only GLSL 1.00 ES (with four extensions), WebGL2 introduces the choice of writing shaders in GLSL version 3.00 ES. While GLSL 3.00 ES introduces important new features, some parts of the language syntax has changed.
 
+These syntax differences between versions can be a complication when writing shader code that is inteded to run both on WebGL1 and WebGL2 environments. In particular when creating reusable code for shader modules, it is desirable to be able to write GLSL code that can work in both GLSL 3.00 and 1.00 shaders.
 
```

Comment:
Should provide link to this article or similar for more information, so interested users can get more details. http://www.shaderific.com/blog/2014/3/13/tutorial-how-to-update-a-shader-for-opengl-es-30